server/api/core/core.js

```diff
@@ -348,22 +369,20 @@ export default {
     //
     // Set Default Roles
     //
-
+    const defaultAdminRoles = ["owner", "admin", "guest", "account/profile"];
     // we don't use accounts/addUserPermissions here because we may not yet have permissions
-    Roles.setUserRoles(accountId, _.uniq(defaultAdminRoles), shopId);
```

Comment:
Was the reason that _.uniq was here because at some point we were allowing `defaultAdminRoles` to be overridden, or perhaps set in `Shops`.  Not sure that ever happened, so this change isn't changing the current behavior, but feels as though something got lost. @impactmass you'll want to be aware of this for implementation of permission groups.